homeassistant/components/vera.py

```diff
@@ -120,13 +124,15 @@ def map_vera_device(vera_device, remap):
 
 
 class VeraDevice(Entity):
-    """Representation of a Vera devicetity."""
+    """Representation of a Vera device entity."""
 
     def __init__(self, vera_device, controller):
         """Initialize the device."""
         self.vera_device = vera_device
         self.controller = controller
-        self._name = self.vera_device.name
+
+        # Append device id to prevent name clashes in HA.
+        self._name = self.vera_device.name + ' ' + str(vera_device.device_id)
```

Comment:
This should not have been added to the name. We should have just updated the entity_id. The name is what we show in the UI and it should not have a device id. We should only assign `self.entity_id` with a unique entity id.

Can someone submit a PR to fix this ?